src/components/messenger/group-management/member-management-menu/container.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+import { RootState } from '../../../../store/reducer';
+import { connectContainer } from '../../../../store/redux-container';
+import { openRemoveMember } from '../../../../store/group-management';
+
+import { MemberManagementMenu } from '.';
+
+export interface PublicProperties {
+  user?: any;
+  canRemove?: boolean;
+
+  onOpenChange: (isOpen: boolean) => void;
+}
+
+export interface Properties extends PublicProperties {
+  activeConversationId: string;
+
+  openRemoveMember: (params: { roomId: string; userId: string }) => void;
+}
+
+export class Container extends React.Component<Properties> {
+  static mapState(state: RootState) {
+    const {
+      chat: { activeConversationId },
+    } = state;
+
+    return {
+      activeConversationId,
+    };
+  }
+
+  static mapActions(_props: Properties): Partial<Properties> {
+    return { openRemoveMember };
```

Comment:
I've added this container and moved `onRemoveMember` saga prop over there, since we're also gonna add more functions here: `setModerator`, `removeModerator` etc.